instrumentation/servlet/servlet-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/servlet/dispatcher/RequestDispatcherAdviceHelper.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.servlet.dispatcher;
+
+import static io.opentelemetry.instrumentation.api.tracer.HttpServerTracer.CONTEXT_ATTRIBUTE;
+
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.SpanContext;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.instrumentation.api.tracer.BaseTracer;
+import io.opentelemetry.instrumentation.servlet.ServletAccessor;
+import io.opentelemetry.javaagent.instrumentation.api.Java8BytecodeBridge;
+
+public class RequestDispatcherAdviceHelper {
+  public static <REQUEST> Context getStartParentContext(
+      ServletAccessor<REQUEST, ?> accessor, REQUEST request) {
```

Comment:
maybe worth passing in requestContextAttribute here, then can delete `JavaxCommonServletAccessor`
```suggestion
  public static <REQUEST> Context getStartParentContext(Object requestContextAttribute) {
```

Reply:
Done.